packages/ui/src/components/atoms/Button.tsx

```diff
@@ -158,10 +159,10 @@ const ButtonStyle = styled.button<ButtonProps>`
 
   &:active {
     box-shadow: 0px 0px 4px 4px rgba(0, 0, 0, 0.4) inset;
-    transition: box-shadow ${buttonAnimationData.duration}
-      ${buttonAnimationData.curve};
+    transition: box-shadow ${buttonAnimationData.duration};
+    ${buttonAnimationData.curve};
```

Comment:
Looks like a typo

Reply:
sorry couldnt understand this, there is only a space difference right?

Reply:
This component has not been changed, it is there in develop branch as well.

Reply:
removed semicolon